docs/_static/WAM_logo_v3.png:Zone.Identifier

```diff
@@ -0,0 +1,4 @@
+[ZoneTransfer]

```

Comment:
This file is related to WSL I believe, you can remove it. I guess it was added to git accidentily.